### ISSUE TITLE
Table filters elements lose focus in High Contrast

### ISSUE THREAD

**Issue by sryeluri** [telerik/kendo-angular#2044]
OS: Windows 1809

Browser: Chrome 70.0.3538.77
Assistive Technology: High Contrast
Steps:
1. Navigate to Application 
2. Press tab until focus is given to the table.
3. Navigate to any filter 
4. Enable High Contrast mode
5. Observe focus indicator.

Actual Results: 
Filter elements do not receive focus in High Contrast

Expected Results:
Focus indicator should not be lost in high contrast

**Comment by ipeshev** [comment]
@sryeluri not sure if I'm reproducing it correctly but focus is lost once you click to enable high contrast , with shift+f11 it looks fine. Tried with Chrome High contrast plugin

**Comment by kraay** [comment]
@ipeshev This bug can be observed on using IE 11. 
This is the image without high contrast mode when focus is on a data cell -
![image](https://user-images.githubusercontent.com/10906811/52937088-2cb0e180-3384-11e9-9b26-542f17ff83e4.png)
On enabling high contrast, the focus indicator is missing and the filter sprite is not available as well. Tabbing doesn't help, and on disabling high contrast, the indicator is visible on the same cell. 
We used the default Windows settings to enable high contrast.
![image](https://user-images.githubusercontent.com/10906811/52937242-9b8e3a80-3384-11e9-908b-be2327ccdb2e.png)

**Comment by tsvetomir** [comment]
@kraay can you please confirm if this issue is related to the project setup in https://github.com/telerik/kendo-ui-core/issues/4856, https://github.com/telerik/kendo-ui-core/issues/4855 and https://github.com/telerik/kendo-ui-core/issues/4854?

**Comment by kraay** [comment]
@tsvetomir Yes, if you run the mock application provided in the above mentioned bugs ([https://runner.telerik.io/fullscreen/oXokIKeS](https://runner.telerik.io/fullscreen/oXokIKeS)) on IE 11 and Enable high contrast in Windows settings, you can repro the issue of the filter sprite not being visible.
